R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
+sglue_data <- function(...) stop(glue::glue_data(..., .envir = parent.frame()), call. = FALSE)
+scollapse <- function(...) stop(glue::collapse(...), call. = FALSE)
```

Comment:
Do we make any collapse calls where `sep` is NOT `"\n"`? It feels prevalent enough that perhaps our wrapper should default to that.